test/controllers/api/v2/context_controller_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class Api::V2::ContextControllerTest < ActionController::TestCase
+  # test_metadata = {
+  #  version: 1,
+  #  user: "admin",
+  #  permissions: %w[perm_0 perm_1 perm_2],
+  # }
+
+  test "should get full metadata" do
+    assert_equal true, true
```

Comment:
I assume these are missing a TODO comment :)

Reply:
Yes, those tests are still missing because I didn't manage to stub/mock the helper method...
Do you have an idea how I might do that?